src/components/Navigation.jsx

```diff
@@ -1,32 +1,98 @@
+      setNavbar(true);
+      setNavText(true);
+      setNavBrand(true);
+    } else {
+      setNavbar(false);
+      setNavText(false);
+      setNavBrand(false);
+    }
+  };
+  //will need to do for logo too
+  useEffect(() => {
+    changeNavbar();
+    window.addEventListener("scroll", changeNavbar);
+  });
 
   return (
-    <Navbar bg="light" expand="lg" sticky="top">
+    <Navbar
+      className="custom-navbar"
+      expand="lg"
+      fixed="top"
+      bg={navbar ? "success" : ""}
+    >
       <Container>
-        <Navbar.Brand href="/">Home Gardnr</Navbar.Brand>
+        <Navbar.Brand href="/">
+          <Image src={navBrand ? brand1 : brand2} alt="Gardnr" width="50px" />
+          {/* <img
+              src={brand}
+              width="50"
+              height="50"
+              className="d-inline-block align-top"
+              alt="React Bootstrap logo"
+            /> */}
+        </Navbar.Brand>
         <Navbar.Toggle aria-controls="basic-navbar-nav flex-grow-1 justify-content-evenly" />
         <Navbar.Collapse id="basic-navbar-nav">
           <Nav className="ms-auto">
-            {/* adds padding of 0.5rem to right side of nav text */}
-            <Navbar.Text className="pe-2">Hello, {user}!</Navbar.Text>
-            <Nav.Link href="/plants">Plant Library</Nav.Link>
-            <Nav.Link href="/my_garden">My Garden</Nav.Link>
-            <Nav.Link href="/weather">Weather Info</Nav.Link>
-
-            <Nav.Link href="/">Logout</Nav.Link>
+            {/* adds padding of 0.5rem to right side of navbar.text */}
+            <Navbar.Text
+              style={navText ? { color: "black" } : { color: "white" }}
+              className="pe-2"
+            >
+              Hello, {user}!
+            </Navbar.Text>
+            <Nav.Link
+              style={navText ? { color: "black" } : { color: "white" }}
+              href="/plants"
+            >
+              Plant Library
+            </Nav.Link>
+            <Nav.Link
+              style={navText ? { color: "black" } : { color: "white" }}
+              href="/my_garden"
+            >
+              My Garden
+            </Nav.Link>
+            <Nav.Link
+              style={navText ? { color: "black" } : { color: "white" }}
+              href="/weather"
+            >
+              Weather Info
+            </Nav.Link>
+            <Nav.Link
+              style={navText ? { color: "black" } : { color: "white" }}
+              href="/"
+            >
+              Logout
+            </Nav.Link>
           </Nav>
         </Navbar.Collapse>
       </Container>
     </Navbar>
   );
 }
+
+//https://dev.to/cesareuseche/react-navbar-change-background-color-on-scroll-react-js-gatsby-2a39#:~:text=To%20create%20the%20rendering%20of,the%20changeBackground%20function%2C%20like%20so.
+
+///  bg="light"
 //questions
 //my plants button/add plants+ (how to do this?)
 //navtext vs disabled - do we want the hello waldo to also be a link?
```

Comment:
I would say only if we implement a proper login/user management route (so you could click on the Hello, Waldo! to go to your user page or something)